Data/core_essentials/ce_kits_familiar_cr.lua

```diff
@@ -0,0 +1,1489 @@
+-- Converted From LST file data\pathfinder\paizo\roleplaying_game\core_essentials\ce_kits_familiar_cr.lst
+-- From repository https://github.com/pcgen/pcgen at commit 11ceb52482855f2e5f0f6c108c3dc665b12af237
+SetSource({
+  SourceLong="Bestiary",
+  SourceShort="B1",
+  SourceWeb="http://paizo.com/store/downloads/pathfinder/pathfinderRPG/v5748btpy8auu",
+  SourceDate="2009-10",
+})
+
+DefineStartingKit({
+  Name="Animal ~ Bat",
+  SourcePage="p.131",
+  Visible=false,
+  Types={
+    "Animal",
+    "Familiar",
+  },
+  Apply=function (kit)
+    kit.SetAlignment({
```

Comment:
This is going to be a huge pain to implement...  This means we need a whole, secondary model that we can pass to these things, that will have all the same methods as the original.  That more or less triples the code for parsing these (since we need to have 2 copies, and a way to reconcile everything.

Reply:
These are "kits" that can be applied to a character one or more times. I don't think these need anything else in the data models, each method is exactly the same as something a user would do.

Reply:
But we need to expose it to the LUA, and it needs to accept the stupid LuaTable objects, and it needs to re-resolve everything. And the model is immutable, but this requires a mutable object.
I'm... not going to implement these (I'm just going to return null).

Reply:
It seems like a better implementation would just be whatever our serialized model that we merge into the current character.  The serialized characters are definitely not going to be LUA files.